src/containers/reports/ducks/selectors.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  AdoptionReport,
+  StewardshipReport,
+  StewardshipReportTableEntry,
+} from './types';
+
+export const getCountAdoptedInPastWeek = (
+  adoptionReport: AdoptionReport | undefined,
+): number | undefined => {
+  return adoptionReport?.adoptionReport.filter(
+    (entry) =>
+      new Date(entry.dateAdopted) >
+      new Date(Date.now() - 7 * 24 * 60 * 60 * 1000),
```

Comment:
Can we have `new Date(Date.now() - 7 * 24 * 60 * 60 * 1000)` as a named constant for readability? Also do we need to use `Date.now()`, and if so, why no `new` keyword? In general we should construct all built-in classes like `Date`.

Reply:
Created a named constant. `Date.now()` returns the number of ms since Jan 1, 1970 while `new Date()` returns the equivalent but as a Date object. We could use a date object in the calculation instead, but then we'd have to do `new Date().getDate()` to get the ms anyway, so I think using `Date.now()` is cleaner.